packages/bundler-plugin-core/src/sentry/telemetry.ts

```diff
@@ -57,16 +56,22 @@ export function addSpanToTransaction(
   return span;
 }
 
-export function captureMinimalError(error: unknown | Error | AxiosError, hub: Hub) {
-  const isAxiosError = error instanceof AxiosError;
-  const sentryError =
-    error instanceof Error
-      ? {
-          name: `${isAxiosError && error.status ? error.status : ""}: ${error.name}`,
-          message: error.message,
-          stack: error.stack,
-        }
-      : {};
+export function captureMinimalError(error: unknown | Error, hub: Hub) {
+  let sentryError;
+
+  if (typeof error === "object") {
+    const e = error as { name?: string; message?: string; stack?: string };
+    sentryError = {
+      name: e.name,
+      message: e.message,
+      stack: e.stack,
+    };
+  } else if (typeof error === "string") {
```

Comment:
Just a though, but what if we just make this a general fallback and handle it as:

```ts
} else {
  sentryError = {
    name: 'Error',
    message: `${error}` // <-- convert anything to string
  };
}
```
}

Reply:
Sounds good to me 👍